### PR TITLE
More rechart examples - Customized Label Pie Chart

### PR DESCRIPTION
**Pull request by samme78**
One more rechart example to help out with #58 

I'm not 100% sure about the style you want. For example this implicit yield

`if x > input.cx then prop.textAnchor.startOfText else prop.textAnchor.endOfText`

Please tell me if there is any changes you want made.

Also, this is my first pull request ever so please tell me if I'm doing something wrong :)

### CODE REVIEW

File: docs/App.fsproj [hunk 1]
```diff
@@ -28,6 +28,7 @@
         <Compile Include="Recharts/BarCharts/MixBarChart.fs" />
         <Compile Include="Recharts/BarCharts/TinyBarChart.fs" />
         <Compile Include="Recharts/BarCharts/PositiveAndNegative.fs" />
+        <Compile Include="Recharts\PieCharts\CustomizedLabelPieChart.fs" />
```
Comment (Shmew) [inline docs/App.fsproj]:
```suggestion
        <Compile Include="Recharts/PieCharts/CustomizedLabelPieChart.fs" />
```

Comment (Zaid-Ajaj) [reply docs/App.fsproj]: Sorry I guess I merged before checking your review. I will change the slashes myself before I publish

File: docs/App.fs [hunk 2]
```diff
@@ -623,6 +624,7 @@ let sidebar (state: State) dispatch =
                 ]
 
                 nestedMenuList "Pie Charts" [
+                    menuItem "Customized Label Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.CustomizedLabelPieChart ]
```
Comment (Shmew) [inline docs/App.fs]:
```suggestion
                    menuItem "Two Level Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.TwoLevelPieChart ]
```

File: docs/App.fs [hunk 3]
```diff
@@ -623,6 +624,7 @@ let sidebar (state: State) dispatch =
...
                     menuItem "Two Level Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.TwoLevelPieChart ]
```
Comment (Shmew) [inline docs/App.fs]:
```suggestion
                    menuItem "Single Angle Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.SingleAngle ]
```

File: docs/App.fs [hunk 4]
```diff
@@ -623,6 +624,7 @@ let sidebar (state: State) dispatch =
...
                     menuItem "Single Angle Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.SingleAngle ]
```
Comment (Shmew) [inline docs/App.fs]:
```suggestion
                    menuItem "Customized Label Pie Chart" [ Urls.Recharts; Urls.PieCharts; Urls.CustomizedLabelPieChart ]
```

**Comment by Zaid-Ajaj** [comment]
> I'm not 100% sure about the style you want. For example this implicit yield

> Please tell me if there is any changes you want made.

The code looks very clean and the implicit yield is exactly how you are supposed to use it! Thank you so much for contributing, really appreciate it!

File: docs/App.fs [hunk 5]
```diff
@@ -672,6 +674,7 @@ let content state dispatch =
     | [ Urls.Recharts; Urls.BarCharts; Urls.PositiveAndNegative ] -> loadMarkdown [ "Recharts" ; "BarCharts"; "PositiveAndNegative.md" ]
     | [ Urls.Recharts; Urls.LineCharts; Urls.BiaxialLineChart ] -> loadMarkdown [ "Recharts"; "LineCharts"; "BiaxialLineChart.md" ]
     | [ Urls.Recharts; Urls.PieCharts; Urls.TwoLevelPieChart ] -> loadMarkdown [ "Recharts"; "PieCharts"; "TwoLevelPieChart.md" ]
+    | [ Urls.Recharts; Urls.PieCharts; Urls.CustomizedLabelPieChart ] -> loadMarkdown [ "Recharts"; "PieCharts"; "CustomizedLabelPieChart.md" ]
```
Comment (Shmew) [inline docs/App.fs]:
```suggestion
   | [ Urls.Recharts; Urls.PieCharts; Urls.SingleAngle ] -> loadMarkdown [ "Recharts"; "PieCharts"; "SingleAngle.md" ]
```

File: docs/App.fs [hunk 6]
```diff
@@ -672,6 +674,7 @@ let content state dispatch =
...
     | [ Urls.Recharts; Urls.PieCharts; Urls.SingleAngle ] -> loadMarkdown [ "Recharts"; "PieCharts"; "SingleAngle.md" ]
```
Comment (Shmew) [inline docs/App.fs]:
```suggestion
    | [ Urls.Recharts; Urls.PieCharts; Urls.CustomizedLabelPieChart ] -> loadMarkdown [ "Recharts"; "PieCharts"; "CustomizedLabelPieChart.md" ]
```

**Comment by Shmew** [comment]
This new suggest change feature is kinda spammy. Everything looks good, Only thing I'd like is to make the order consistent.

**Comment by Zaid-Ajaj** [comment]
@Shmew your suggestions were on point, I included them all (I think) and published the docs website. @samme78 The example should be online in no time, thanks again and I hope you keep 'em coming :smile:

**Pull request merged**